keeper-gradle-plugin/src/main/java/com/slack/keeper/ZipFlingerExt.kt

```diff
@@ -30,7 +30,18 @@ internal fun File.classesSequence(): Sequence<Pair<String, File>> {
     .filter { it.extension == "class" }
     .filterNot { "META-INF" in it.name }
     .sortedBy { it.invariantSeparatorsPath }
-    .map { it.absolutePath.removePrefix(prefix).removePrefix("/") to it }
+    .map {
+      // zip specification "4.4.17.1 file name: (Variable)" items:
```

Comment:
Have you considered extracting a (private) extension function/property? e.g. `File.zipSpecFriendlyPath` with a potentially better name? That way there would be a place to put a link to the doc, explanation etc.

Reply:
Hey Robert, I'm handling the review. Your help is appreciated but ultimately not necessary here, thanks! This is fine with me as-is.